### ISSUE TITLE
Screen Reader announcing "button collapsed" for tabster dom element

### ISSUE THREAD

**Issue by saahilchawande123** [microsoft/tabster#260]
Hello,
We are using tabster in our project and it seems that tabster is adding a dom element at the bottom of the page which causes the screen reader to read as "button collapsed". 

Screenshot for the DOM element being rendered:
<img width="958" alt="Code snippet _ Screen reader is announcing as button collapsed" src="https://user-images.githubusercontent.com/116312427/227660005-c11669e2-7b73-4b60-b68c-10a4ec6530fb.png">

Screen Recording for the occurring issue:

https://user-images.githubusercontent.com/116312427/227660340-56bf33bc-af36-41cb-8c64-b335abaf4574.mp4

Additional Notes:
I made sure that the mentioned DOM element added by tabster is causing the issue by deleting the element from DOM (from the dev tools) and the screen reader does not read "button collapsed".

Do let me know if more information is needed from my side.

**Comment by saahilchawande123** [comment]
Can I please get an update on this one?

**Comment by mshoho** [comment]
This element is not added by Tabster. It just has Tabster attribute which is added by your component library or application code, but the element itself is not added by Tabster. Which component library are you using? I would suggest asking there.
